src/week.jsx

```diff
@@ -41,6 +42,12 @@ export default class Week extends React.Component {
     }
   }
 
+  handleWeekClick = (day, weekNumber, event) => {
+    if (this.props.onWeekSelect) {
+      this.props.onWeekSelect(day, weekNumber, event)
+    }
+  }
+
```

Comment:
This method is here so we can provide the week number along with the first day of week to the callback.